site/docs/master/upgrade-to-1.2.md

```diff
@@ -98,4 +98,4 @@ _Note: if you're upgrading from v1.0, follow the [upgrading to v1.1][2] instruct
 [0]: https://github.com/vmware-tanzu/velero/releases/tag/v1.1.0
 [1]: https://github.com/vmware-tanzu/velero/releases/tag/v1.0.0
 [2]: https://velero.io/docs/v1.1.0/upgrade-to-1.1/
-[3]: install-overview.md#install-the-cli
+[3]: advanced-installation.md#install-the-cli
```

Comment:
"Install the CLI" is on the getting-started page

Reply:
Yeah, I think so too!